### ISSUE TITLE
Proxmox: Web UI, project closes automaticly after short time.

### ISSUE THREAD

**Issue by mkono87** [GNS3/gns3-web-ui#1269]
Been running GNS3 in proxmox for a while using the desktop clients without issues.

Decided to start testing/using the webui and once I start appliances, after a minute or so the console will disappear. Not understanding what just happened, I click the console option again in which I presented with a message at the bottom  stating the project is not open. I have to usually exit the project and enter it again. I do not see this issue using the desktop client.

**Comment by piotrpekala7** [comment]
@mkono87 thanks for creating this task, could you please add more information about browser (with version) & OS you use?

**Comment by piotrpekala7** [comment]
could you also tell also which appliance you have in project?

**Comment by mkono87** [comment]
@piotrpekala7 
I am using Firefox on both Windows and Ubuntu based OS's. (Version 98) I have tried both Cisco vIOS and IOL images. Both L2 and L3.
Il test another browser shortly and report back.

**Comment by mkono87** [comment]
@piotrpekala7 

Seems to be okay on edge however I cant open up my console app (solar-putty) from edge but thats not a gns3 issue. Brave browser on Linux was just lagging like crazy however.

**Comment by dpnetca** [comment]
Think I am hitting this same issue, here is a bit more information from what I have found.  also using Proxmox, tried it with GNS3 deployed in 3 different ways (imported from OVA, imported from KVM and installed in in Ubuntu I think using this method: https://docs.gns3.com/docs/getting-started/installation/remote-server/#!

Initially noticed this with 2.2.32, patched to 2.2.34 and same issue still occurs

load GNS web gui in firefox, open a project, start the project. and it starts fine.  If I open putty and telnet to GNS3-ip:5000 or whatever port is assigned to console for device it connects fine and everything runs fine.  if I right click the device and web console, again no issues.  what causes it to close though is if i right click and console load the link using the GNS3+telnet link which hits the GNS3 WebClient it launches the terminal fine, runs for 1-2 seconds then the terminal closes (I see exit show up in terminal briefly) and clicking on project generates an error that the project is not open.

Using Edge Chromium and things are fine and stable

in the GNS3 WebClient I have tried setting console to use Putty or Putty standalone, both have the same issue.  I have tried launching Putty Standalone with from command prompt using the exact same syntax and that works fine no crash.  it only seems when clicking console from Firefox and having it launch the link with the GNS3 webclient.

Initially when I noticed it i had 2 Nexus9000v and a docker image (the python, Go, ....something one)  for future tests I tried a couple different images usually a docker image as they load quick, with same behavior
